README.md

```diff
@@ -76,21 +77,45 @@ It's also possible to build and use a locally built image. See the
 
 ## Securing the Docker Compose ACAP using TLS
 
-The Docker Compose ACAP can be run either unsecured or in TLS mode. The Docker Compose ACAP uses
-TLS as default. Use the "Use TLS" dropdown in the web interface to switch
-between the two different modes. It's also possible to toggle this option by
-calling the parameter management API in [VAPIX](https://www.axis.com/vapix-library/) and setting the
-`root.dockerdwrapperwithcompose.UseTLS` parameter to `yes` or `no`. The following commands would
-enable TLS:
+The Docker Compose ACAP can be run in either TLS mode or unsecured mode. The Docker Compose ACAP
+uses unsecured mode by default. There is an option to choose between "TCPSocket" and "IPCSocket" socket
+parameters. The API listens to IPC socket by default, even if the "IPCSocket" parameter is set to 'no'.
```

Comment:
This is true if neither TCP or IPC socket is set. If TCP socket is set I believe you need to acitvely add the IPC socket for it to be active. 
I don't think you should talk so much about the IPC socket in this section that has to do with TLS, since it does not apply to it.